www-client/seamonkey/seamonkey-2.53.14.ebuild

```diff
@@ -0,0 +1,559 @@
+
+DESCRIPTION="Seamonkey Web Browser"
+HOMEPAGE="https://www.seamonkey-project.org/"
+
+LICENSE="MPL-2.0 GPL-2 LGPL-2.1"
+SLOT="0"
+SYSTEM_IUSE=( +system-{av1,harfbuzz,icu,jpeg,libevent,libvpx,png,sqlite} )
+IUSE="+chatzilla cpu_flags_arm_neon dbus +gmp-autoupdate +ipc jack
+lto pulseaudio selinux startup-notification test webrtc wifi"
+IUSE+=" ${SYSTEM_IUSE[@]}"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	app-arch/unzip
+	app-arch/zip
+	>=dev-lang/nasm-2.13
+	dev-lang/perl
+	>=sys-devel/binutils-2.16.1
+	virtual/pkgconfig
+	<=virtual/rust-1.63.0
+	amd64? ( >=dev-lang/yasm-1.1 )
+	lto? ( sys-devel/binutils[gold] )
+	x86? ( >=dev-lang/yasm-1.1 )
+"
+COMMON_DEPEND="
+	app-arch/bzip2
+	>=app-text/hunspell-1.5.4:=
+	dev-libs/atk
+	>=dev-libs/glib-2.26:2
+	>=dev-libs/libffi-3.0.10:=
+	>=dev-libs/nspr-4.23
+	>=dev-libs/nss-3.47.1
+	media-libs/fontconfig
+	>=media-libs/freetype-2.4.10
+	>=media-libs/mesa-10.2:=
+	>=sys-libs/zlib-1.2.3
+	>=x11-libs/cairo-1.10[X]
+	x11-libs/gdk-pixbuf
+	>=x11-libs/gtk+-3.4.0:3
+	x11-libs/libX11
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/libxcb:=
+	>=x11-libs/pango-1.22.0
+	x11-libs/pixman
+	media-video/ffmpeg
+	virtual/freedesktop-icon-theme
+	dbus? (
+		>=dev-libs/dbus-glib-0.72
+		>=sys-apps/dbus-0.60
+	)
+	jack? ( virtual/jack )
+	kernel_linux? ( !pulseaudio? ( media-libs/alsa-lib ) )
+	pulseaudio? ( || (
+		media-sound/pulseaudio
+		>=media-sound/apulse-0.1.9
+	) )
+	startup-notification? ( >=x11-libs/startup-notification-0.8 )
+	system-av1? (
+		>=media-libs/dav1d-0.3.0:=
+		>=media-libs/libaom-1.0.0:=
+	)
+	system-harfbuzz? (
+		>=media-gfx/graphite2-1.3.9-r1
+		>=media-libs/harfbuzz-1.3.3:0=
+	)
+	system-icu? ( >=dev-libs/icu-59.1:= )
+	system-jpeg? ( >=media-libs/libjpeg-turbo-1.2.1 )
+	system-libevent? ( >=dev-libs/libevent-2.0:0= )
+	system-libvpx? ( >=media-libs/libvpx-1.8.0:0=[postproc] )
+	system-png? ( >=media-libs/libpng-1.6.31:0=[apng] )
+	system-sqlite? ( >=dev-db/sqlite-3.38.2:3[secure-delete] )
+	wifi? (
+		kernel_linux? (
+			>=dev-libs/dbus-glib-0.72
+			net-misc/networkmanager
+			>=sys-apps/dbus-0.60
+		)
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	selinux? ( sec-policy/selinux-mozilla )
+"
+DEPEND="${COMMON_DEPEND}
+	amd64? ( virtual/opengl )
+	x86? ( virtual/opengl )
+"
+
+# allow GMP_PLUGIN_LIST to be set in an eclass or
+# overridden in the enviromnent (advanced hackers only)
+[[ -z ${GMP_PLUGIN_LIST} ]] && GMP_PLUGIN_LIST=( gmp-gmpopenh264 gmp-widevinecdm )
+
+BUILD_OBJ_DIR="${S}/seamonk"
+
+pkg_setup() {
+	if [[ ${PV} == *_beta* ]] || [[ ${PV} == *_pre* ]] ; then
+		ewarn
+		ewarn "You're using an unofficial release of ${PN}. Don't file any bug in"
+		ewarn "Gentoo's Bugtracker against this package in case it breaks for you."
+		ewarn "Those belong to upstream: https://bugzilla.mozilla.org"
+	fi
+
+	if ver_test $(rustc -V | cut -d" " -f2) -ge "1.64"; then
```

Comment:
Check also the follow-up commit I linked in the previous comment to fix this with "nightly".

Reply:
Bussy IRL, so, sorry it took a bit long to reply.

cbindgen BDEPEND is in the last commit and changed the rust check to handle the "nightlies".

Reply:
Did some test on a test system yesterday, where I updated glibc to 2.36 and tried to reproduce the issue, but that didn't work. So, it does not seem to be glibc-2.36 related to me. I had a conversation with upstream regarding this issue, they think it is more likely due to using the system hunspell, instead of the bundled one. Anyway, they suggested to change the include, as it was wrong in the first place (it gets copied to a mozilla/ dir by the build file) and see if that fixes it.